software/litlyx-analytics.yml

```diff
@@ -0,0 +1,12 @@
+
+name: "Litlyx"
+website_url: "https://litlyx.com"
+source_code_url: "https://github.com/Litlyx/litlyx"
+description: "All-in-one Analytics Solution. Setup in 30 seconds. Display all your data on an AI-powered dashboard. Fully self-hostable and GDPR compliant."
+licenses:
+  - Apache-2.0
+platforms:
+  - Nodejs
```

Comment:
```suggestion
```
Self-hosting docs only show docker as a possible option to self-host this tool
https://docs.litlyx.com/self-hosting/introduction

Reply:
![image](https://github.com/user-attachments/assets/30ca3fbf-ad32-4f48-b8bf-8050ee878cab)
you can, its in the repo description